main/src/main/scala/sbt/Cross.scala

```diff
@@ -158,8 +158,8 @@ object Cross {
 
         // Execute using a blanket switch
         projCrossVersions.toMap.apply(extracted.currentRef).flatMap { version =>
-          // Force scala version
-          Seq(s"$SwitchCommand $verbose $version!", aggCommand)
+          // Set scala version
+          Seq(s"$SwitchCommand $verbose $version", aggCommand)
```

Comment:
I think this ! is necessary for sbt 0.13 emulation - https://github.com/sbt/sbt/issues/5497#issuecomment-618740983